packaging/installer/README.md

```diff
@@ -6,150 +6,8 @@ Netdata is very flexible and can be used to monitor all kinds of infrastructure.
-
-### Make your deployment production ready
-
-Go through our [deployment guides](/docs/deployment-guides/README.md), for suggested configuration changes for production deployments.
-
-## Advanced installation options and troubleshooting
-
-### Automatic updates
-
-By default, Netdata's installation scripts enable automatic updates for both nightly and stable release channels.
-
-If you preferred to update your Netdata Agent manually, you can disable automatic updates by using the `--no-updates`
-option when you install or update Netdata using the [automatic one-line installation script](/packaging/installer/methods/kickstart.md).
-
-```bash
-wget -O /tmp/netdata-kickstart.sh https://get.netdata.cloud/kickstart.sh && sh /tmp/netdata-kickstart.sh --no-updates
-```
-
-With automatic updates disabled, you can choose exactly when and how you [update Netdata](/packaging/installer/UPDATE.md).
-
-### Nightly vs. Stable Releases
-
-**Nightly**: We create nightly builds every 24 hours. They contain fully-tested code that fixes bugs or security flaws,
-or introduces new features to Netdata. Every nightly release is a candidate for then becoming a stable release—when
-we're ready, we simply change the release tags on GitHub. That means nightly releases are stable and proven to function
-correctly in the vast majority of Netdata use cases. That's why nightly is the _best choice for most Netdata users_.
-
-**Stable**: We create stable releases whenever we believe the code has reached a major milestone. Most often, stable
-releases correlate with the introduction of new, significant features. Stable releases might be a better choice for
-those who run Netdata in _mission-critical production systems_, as updates will come more infrequently, and only after
-the community helps fix any bugs that might have been introduced in previous releases.
-
-**Pros of using nightly releases:**
-
-- Get the latest features and bug fixes as soon as they're available
-- Receive security-related fixes immediately
-- Use stable, fully-tested code that's always improving
-- Leverage the same Netdata experience our community is using
-
-**Pros of using stable releases:**
-
-- Protect yourself from the rare instance when major bugs slip through our testing and negatively affect a Netdata installation
-- Retain more control over the Netdata version you use
-
-### Anonymous statistics
-
-Starting with v1.30, Netdata collects anonymous usage information by default and sends it to a self-hosted PostHog instance within the Netdata infrastructure. Read about the information collected, and learn how to-opt, on our [anonymous statistics](/docs/netdata-agent/configuration/anonymous-telemetry-events.md) page.
-
-The usage statistics are _vital_ for us, as we use them to discover bugs and prioritize new features. We thank you for
-_actively_ contributing to Netdata's future.
```

Comment:
This _NEEDS_ to be kept, period. Users should not have to go digging to find out that we do anonymous statistics collection.

Reply:
I added this back in the file